support/android/apk/servoapp/src/main/java/org/servo/servoshell/MainActivity.java

```diff
@@ -138,6 +144,34 @@ public void onStopClicked(View v) {
         mServoView.stop();
     }
 
+    @Override
+    public void onImeShow() {
+        InputMethodManager imm = getSystemService(InputMethodManager.class);
```

Comment:
I'm not sure what is the recommended approach, but  in `setupUrlField` we use `getSystemService(Context.INPUT_METHOD_SERVICE);`. Should one of them be changed so that these are consistent?